modules/indicator.py

```diff
@@ -0,0 +1,15 @@
+from typing import Any
+
+class Indicator:
+
+    def __init__(self, type: str, content: Any, domain: str | None = None):
+        self.type = type
+        self.content = content
+        self.domain = domain
```

Comment:
`self.domain` may not be needed - it's here but only ended up being used in one function.